docs/install.md

```diff
@@ -3,120 +3,307 @@ SPDX-FileCopyrightText: The RamenDR authors
 SPDX-License-Identifier: Apache-2.0
 -->
 
-# Install
+# Installing Ramen
+
+This guide describes how to install Ramen's disaster recovery operators on your
+cluster environment.
+
+## Overview
+
+Ramen consists of two operators that work together to provide disaster recovery
+capabilities:
+
+1. **Ramen Hub Operator** - Installed on the OCM hub cluster, orchestrates DR
+   operations across managed clusters
+1. **Ramen DR Cluster Operator** - Installed on each OCM managed cluster,
+   manages local volume replication and workload protection
 
 ## Prerequisites
 
-### OCM managed multi-cluster setup
+Before installing Ramen, ensure the following requirements are met:
+
+### 1. Kubernetes Versions
+
+**Supported versions:**
+
+- Kubernetes 1.20 or higher
+
+### 2. Open Cluster Management (OCM) Setup
+
+Ramen requires an
+[OCM](https://open-cluster-management.io/docs/concepts/architecture/) hub cluster
+with at least two managed clusters for disaster recovery operations.
+
+**Requirements:**
+
+- OCM hub cluster with `multicluster-engine` installed
+- At least 2 managed clusters registered with the hub
+- All clusters must be able to communicate with each other
+
+For OCM installation instructions, see the [OCM installation guide](https://open-cluster-management.io/docs/getting-started/installation/).
+
+### 3. Storage Replication Support
+
+Ramen supports two disaster recovery modes, each with different storage requirements:
+
+#### Sync Mode (Metro DR)
+
+Sync mode uses a dedicated external storage cluster that all managed clusters
+connect to, enabling synchronous, real-time data replication.
+
+**Supported:**
+
+- Any CSI-compatible storage that supports synchronous replication
```

Comment:
Reword to "Any CSI-compatible storage that supports connecting to a shared external storage cluster"